client/gutenberg/extensions/map/mapbox-map-formatter/index.js

```diff
@@ -0,0 +1,25 @@
+/** @format */
+
+/* TODO: Replace with styles created in A8C-owned account */
+export function mapboxMapFormatter( map_style, map_details ) {
+	const style_urls = {
+		default: {
+			details: 'mapbox://styles/rabberson/cjnxjme2m054v2rs2gf3nggx6',
+			no_details: 'mapbox://styles/rabberson/cjnxl6xqp55kt2sk55ivay8r0',
+		},
+		black_and_white: {
+			details: 'mapbox://styles/rabberson/cjnxjofik08ax2rmx47rfzhji',
+			no_details: 'mapbox://styles/rabberson/cjnxjvlk108kq2slmjnd9beni',
+		},
+		satellite: {
+			details: 'mapbox://styles/mapbox/satellite-streets-v10',
+			no_details: 'mapbox://styles/mapbox/satellite-v9',
+		},
+		terrain: {
+			details: 'mapbox://styles/rabberson/cjnxl0upd52ik2srtr0zh216f',
+			no_details: 'mapbox://styles/rabberson/cjnxl28cc54rl2roaamgb621y',
```

Comment:
Probably good to get these to a shared a8c account before bundling to Jetpack even as a beta version for all the world, since this would live there like _forever_.

Reply:
Yes, absolutely. After we merge this PR I'll open a new one that does nothing but replace these styles. That will be a diff we can all get behind.